user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -4,91 +4,205 @@ uid: DashboardGrid
 
-## Using the grid component in read mode
+- By default, the number of displayed columns and rows is set to *Auto* (i.e. all columns and/or rows are displayed). To specify a fixed number, clear the checkbox and enter the desired value(s). To revert to *Auto*, delete the entry.
 
-- In read mode, you can manipulate the grid component to **navigate through the columns and rows**:
+- To switch between scaling options, click the icon next to the column or row count:
 
-  - If the number of items exceeds the size of the component, a scrollbar appears when you hover over the component, allowing you to navigate through the items.
+  - ![Scaled to fit (fixed)](~/user-guide/images/Fixed.png) : The cells are scaled dynamically to fit the grid.
 
-  - From DataMiner 10.3.0 [CU11]/10.4.2 onwards<!--RN 38191-->, when you are using a mobile device:
+  - ![Scaled to fit (scaling)](~/user-guide/images/Scaling.png) : The cells have a fixed size. This is the default setting.
 
-    - You can move the grid left or right and up or down by sliding one finger across the component.
+> [!NOTE]
+> The number of items that can be displayed in a grid component is limited to 1000<!--RN 37699-->.
 
-    - You can select grid items by tapping them.
+#### Customizing grid items
 
-- From DataMiner 10.4.0 [CU13]/10.5.0 [CU1]/10.5.4 onwards<!--RN 42322-->, when you **select a cell in the grid**, it will by default be highlighted with a blue border and a light-blue background. This can for instance be useful when the grid's [component data](xref:Component_Data) (i.e. *Components* > *Grid #* > *Selected items* > *Tables*) is used in another component, clearly indicating which data is driving the content in the linked component.
+Using the Template Editor, you can fully customize the appearance of each grid item (or block). Thanks to [conditional formatting](xref:Template_Editor#adding-conditional-cases-to-a-layer), **each item can be styled differently** to highlight key information.
+
+To edit the grid item template:
+
+1. In the *Layout* pane, navigate to the *Item templates* section.
+
+   This section shows a preview of the template currently applied to the grid component.
+
+1. Click *Edit* to open the Template Editor.
+
+1. Make your changes as described under [Using the Template Editor](xref:Template_Editor).
+
+<div style="display: flex; align-items: center; justify-content: space-between; margin: 0 auto; max-width: 100%;">
+  <div style="border: 1px solid #ccc; border-radius: 10px; padding: 10px; flex-grow: 1; background-color: #DEF7FF; margin-right: 20px; color: #000000;">
+    <b>💡 TIPS TO TAKE FLIGHT</b><br>
+    More of a visual learner? Watch the <a href="https://community.dataminer.services/courses/kata-16/" style="color: #657AB7;"><i>Introducing grid components</i> video tutorial</a> to see the Template Editor at work!
+  </div>
+  <img src="~/images/Skye.svg" alt="Skye" style="width: 100px; flex-shrink: 0;">
+</div>
+
+<br>
+
+Some **real-life examples**:
```

Comment:
These are not the prettiest examples I've seen. It can be merged as is but perhaps you can check with Rens to get some better examples.